### PR TITLE
Compilation error/warning for wdog imx driver

### PR DESCRIPTION
**Pull request by clementfaure**

### CODE REVIEW

File: core/drivers/imx_wdog.c [hunk 1]
```diff
@@ -154,18 +154,16 @@ static TEE_Result imx_wdog_base(vaddr_t *wdog_vbase)
 	*wdog_vbase = (vaddr_t)phys_to_virt(WDOG_BASE, MEM_AREA_IO_SEC, 1);
 #if defined(CFG_IMX_WDOG_EXT_RESET)
 	ext_reset_output = true;
+#else
+	ext_reset_output = false;
```
Comment (jforissier) [inline core/drivers/imx_wdog.c:158]: Global variables are always initialized to `false` by default, so this is not desirable. I'd much prefer you add `__maybe_unused` to the definition of `ext_reset_output`.

**Comment by clementfaure** [comment]
Should I wrap the compilation error message in the commit message ?

**Comment by jforissier** [comment]
> Should I wrap the compilation error message in the commit message ?

I prefer not. Checkpatch has a special rule to match compiler messages, but this syntax (` | <some text>`) is not handled yet it seems...

**Review by jforissier — COMMENTED** [review]
`Acked-by: Jerome Forissier <jerome@forissier.org>`

Two minor comments:
- A better prefix may be `core: drivers: imx_wdog:` for both commits
- "compilation warning" not "warning compilation" in the second commit subject

**Comment by clementfaure** [comment]
@jforissier Tags applied
- I've a slight preference for `drivers: imx_wdog` rather than `core: drivers: imx_wdog`. I hope it's fine for you.
- Fixed typo

**Pull request merged**